packages/forms-web-app/src/pages/projects/section-51/_translations/cy.test.js

```diff
@@ -0,0 +1,18 @@
+const section51Translations__CY = require('./cy.json');
+
+describe('pages/projects/section-51/_translations/cy', () => {
+	it('should return the welsh projects translations', () => {
+		expect(section51Translations__CY).toEqual({
+			adviceTo: 'Cyngor i',
+			anonymous: 'Dienw',
+			dateAdviceGiven: 'Dyddiad y rhoddwyd y cyngor',
+			dateOfMeeting: 'Dyddiad y cyfarfod',
+			enquiryFrom: 'Ymholiad gan',
+			enquiryType: 'Math o ymholiad',
+			from: 'Oddi wrth',
+			meetingWith: 'Cyfarfod â',
+			viewAdviceTo: 'Gweld cyngor i',
+			viewMeetingWith: 'Gweld cyfarfod gyda'
+		});
+	});
+});
```

Comment:
I wonder about the value of these tests, since they're really just reiterating what's in the JSON files. It is the standard across the repo though so not something you need to change.